_episodes/03-packaging-installing.md

```diff
@@ -370,24 +288,108 @@ from a local directory.
 > {: .output}
 {: .callout}
 
-So, to install our package, we can run:
+So, to install our package, we can use **Pip**. [Pip](https://pypi.org/project/pip/) is a package manager,
+that is, a package that install packages. We are going to learn more about it in the next section.
+For now, we can run:
 
 ~~~
 cd conversions
 pip install -e .
 ~~~
 {: .language-bash}
-The `-e` flag (aka `--editable`) tells pip to install this package in editable mode. This allows us to make 
+The `-e` flag (aka `--editable`) tells Pip to install this package in editable mode. This allows us to make 
 changes to the package without re-installing it. Analysis code can change dramatically over time, so this is a 
 useful option!
 
+## Pip
+
+Pip is the most common package manager for python. Pip allows you to easily install python packages locally from your computer or from an online repository like the [Python Package Index (PyPI)](https://pypi.org/). Once a package is installed with pip, you can `import` that package and use it in your own code.
```

Comment:
pypi is mentioned above when introducing setup.py

I think that's why i initially had this section where it is currently

Reply:
Thanks for noticing this.

I pushed a5be1e3 that edits the phrasing to properly introduce Pip.